server/http/interceptors/interceptors.go

```diff
@@ -273,18 +273,18 @@ func (w *instrumentedResponseWriter) Flush() {
 	}
 }
 
-func alertOnPanic() {
+func alertOnPanic(err any) {
```

Comment:
👀

Reply:
Hope this was self-explanatory. We were previously not logging the panic message.